src/components/hacs-repository-card.ts

```diff
@@ -361,7 +361,7 @@ export class HacsRepositoryCard extends LitElement {
 
         .status-border {
           border-style: solid;
-          border-width: 1px;
+          border-width: min(var(--ha-card-border-width, 1px), 10px);
```

Comment:
this should be max right?
did you test this?

Reply:
No, it's min. By default it's 1px, eval process:
```
          border-width: min(var(--ha-card-border-width, 1px), 10px);
          border-width: min(1px, 10px);
          border-width: 1px;
```
If someone makes their theme have thicker borders (example: 3px):
```
          border-width: min(var(--ha-card-border-width, 1px), 10px);
          border-width: min(3px, 10px);
          border-width: 3px;
```
If someone overdoes their theme:
```
          border-width: min(var(--ha-card-border-width, 1px), 10px);
          border-width: min(50px, 10px);
          border-width: 10px;
```
Tested in browser console.

Reply:
Cool 👍

Reply:
Again, it's the theme's decision.